src/client/e2e/credit.spec.ts

```diff
@@ -59,27 +56,28 @@ test.describe("Credit", () => {
 
       // Navigate to Live
       await rfqsPage.getByText(/Live/).click()
-      await rfqsPage.waitForTimeout(15000)
 
-      await rfqsPage
-        .locator("[data-testid='quotes']")
+      const firstquote = await rfqsPage
```

Comment:
Minor typo

Reply:
@SHolleworth agree thanks, I've corrected it

Reply:
@SHolleworth can you review again for approval ? thanks